texar/modules/classifiers/classifier_base.py

```diff
@@ -40,3 +40,7 @@ def default_hparams() -> Dict[str, Any]:
 
     def forward(self, *input: torch.Tensor):  # pylint: disable=redefined-builtin
         raise NotImplementedError
+
+    @property
+    def output_size(self) -> int:
```

Comment:
I don't think it's necessary to add this implementation as long as the class is an `ABC`. Does it fail the CI otherwise?